tests/functional/ovnnorthd_controller_test.go

```diff
@@ -335,7 +197,7 @@ var _ = Describe("OVNNorthd controller", func() {
 				Name:      "ovn-northd",
 			}
 			th.SimulateDeploymentReadyWithPods(
-				deploymentName, map[string][]string{namespace + "/internalapi": {"10.0.0.1"}},
```

Comment:
This crashes in test run; I think because the controller no longer fills in the annotations, so they stay empty after create. Then lib-common assumes that SOME annotations are defined, and tries to write to `nil` map.

I think it's probably a bug in lib-common helper (it shouldn't assume IP addresses); we can fix it in lib-common, but in the meantime, you can also carry your own simulate helper that would not access nil-annotations.

If you go the in-house workaround path, please make sure you link to the PR / issue reported for lib-common, so that we can clean up later.